docs/cheaha/migration_overview.md

```diff
@@ -0,0 +1,194 @@
+# Cheaha Data Migration
+
+## User Impact Summary
+
+- All user and project data will be migrated on a rolling basis over the coming weeks
+- Users will receive two day notice before their account is migrated
+- During migration, access to Cheaha will be revoked and all jobs will be requeued and held
+- Plan to lose access to Cheaha for up to **2 days**
+- Once migration is complete, account certification will be required through the [web portal](https://rc.uab.edu)
+
+## Potential Sections of Interest
+
+- [Migraton Procedure](#migration-procedure)
+- [General Timeline](#general-timeline)
+- [Summary of Available Compute During the Migration](#gpfs-5-compute-nodes)
+
+## Overview
+
+Research Computing will be performing a cluster migration for all data on Cheaha from our current GPFS 4 storage system to our new GPFS 5 storage system over the coming weeks. This migration is necessary due to GPFS 4 reaching end-of-life and losing vendor support as well as allowing us to perform further, necessary upgrades to Cheaha over the coming months and years. This is a large scale migration covering [INSERT NUMBER OF FILES] files and [INSERT NUMBER OF PiB] petabytes of data, and so to facilitate a migration of this size, a more intricate process was necessary beyond a whole-cluster shutdown and move. This page is to provide you with information covering the migration plan, our rationale behind certain decisions, and how your account will be affected pre- and post-migration.
+
+### Batching Accounts Into Communities
+
+In order to reliably replicate user data during the migration, it's imperative that those data are quiescent. While a whole-cluster shutdown would address that, the estimated downtime would be overly burdensome to all researchers. Instead, we have split the Cheaha userbase into a number of communities (batches) based on shared access to project spaces. This means for each project space, to the best of our ability, we will be migrating every member of that project at the same time. Our goal is to minimize both individual account downtime as well as overall migration time while making sure each researcher has access to the data they need pre- and post-migration.
+
+<!-- markdownlint-disable MD046 -->
+!!! important
+    It was not possible to make sure all users were transferred with all of their projects while also keeping community sizes manageable. We have minimized the number of people this affects as much as possible. We will notify users if they are part of groups being migrated in a different community when that occurs.
+<!-- markdownlint-enable MD046 -->
+
+## Migration Procedure
+
+The following procedure is performed on a **per-community** basis.
```

Comment:
If this is the Research-facing facet of the procedure, let's rephrase to "Each community, or batch of user groups, should expect the following procedure when it is their turn to migrate."

Reply:
To me, community implies a level of similarity among its members higher than batch. The similar quality we are creating our batches from is group membership as opposed to random assignment that I personally think is associated with batch. I have changed the wording to be similar to what William has suggested though